modules/sensors/lm75.c

```diff
@@ -81,7 +81,7 @@ void vTaskLM75( void* Parameters )
                 check_sensor_event(temp_sensor);
             }
         }
-        vTaskDelay(xFrequency);
+        vTaskDelay(pdMS_TO_TICKS(xFrequency));
```

Comment:
This is not the correct place to use the `pdMS_TO_TICKS` macro, you should use it here instead:
https://github.com/lnls-dig/openMMC/blob/93f1048aacaa84b7b9f98de7f4e56eec4c8a678c/modules/sensors/lm75.c#L49